sonar/modules/documents/jsonschemas/documents/document-v1.0.0_src.json

```diff
@@ -125,41 +125,45 @@
     },
     "organisation": {
       "title": "Organisation",
-      "description": "Organisation linked to document.",
       "type": "object",
       "properties": {
         "$ref": {
           "title": "Organisation",
           "type": "string",
           "pattern": "^https://sonar.ch/api/organisations/.*?$",
-          "minLength": 1
+          "minLength": 1,
+          "form": {
+            "remoteOptions": {
+              "type": "organisations"
+            }
+          }
         }
       },
       "required": [
         "$ref"
-      ]
+      ],
+      "form": {
+        "hide": true,
+        "navigation": {
+          "essential": true
+        }
+      }
     },
     "documentType": {
       "$ref": "type-v1.0.0.json"
     },
     "title": {
-      "title": "Title",
+      "title": "Titles",
```

Comment:
Is 'titles' field isn't an essential field ? It seems not be assigned to 'essential' but maybe required field doesn't need this attribute ?

Reply:
It's not mandatory to put it into essentials, because this field is always available in the form. Essential is just for fields that could be removed from editor.